pallets/gear-scheduler/src/tests.rs

```diff
@@ -73,7 +73,8 @@ fn populate_wl_from(
         .expect("Failed to insert task");
     WaitlistOf::<Test>::insert(dispatch, u64::MAX).expect("Failed to insert to waitlist");
     Balances::reserve(&src, DEFAULT_GAS as u128).expect("Cannot reserve gas");
-    GasHandlerOf::<Test>::create(src, mid, DEFAULT_GAS).expect("Failed to create gas handler");
+    GasHandlerOf::<Test>::create(src, mid.into(), DEFAULT_GAS)
```

Comment:
What the reason to use `.into()` now everywhere ?

Reply:
From my point of view this a necessity from now

Reply:
Because we introduced `GasNodeId`